src/utils/Cypher.ts

```diff
@@ -136,11 +137,12 @@ export default class Cypher {
   private static async getCryptoSetting(tenantID: string): Promise<CryptoSetting> {
     const cryptoSettings = (await SettingStorage.getCryptoSettings(tenantID)).crypto;
     if (!cryptoSettings) {
+      const tenantName = (await TenantStorage.getTenant(tenantID)).name;
       throw new BackendError({
         source: Constants.CENTRAL_SERVER,
         module: MODULE_NAME,
         method: 'getCryptoSetting',
-        message: `Tenant with ID: ${tenantID} does not have crypto settings.`
+        message: `Tenant with ID: ${tenantName} does not have crypto settings.`
```

Comment:
Please read log message you change and ensure the information given are understandable : if you switch to tenant name, it's no more the tenantID. 
I think you can log both `Tenant ${tenantName} (tenandID: ${tenantID}) does not have crypto settings`.